plugins/auth-backend/src/providers/saml/provider.ts

```diff
@@ -119,14 +119,14 @@ export type SamlProviderOptions = {};
 export const createSamlProvider = (
   _options?: SamlProviderOptions,
 ): AuthProviderFactory => {
-  return ({ providerId, globalConfig, config, tokenIssuer, logger }) => {
+  return ({ providerId, globalConfig, config, tokenIssuer }) => {
     const opts = {
       callbackUrl: `${globalConfig.baseUrl}/${providerId}/handler/frame`,
       entryPoint: config.getString('entryPoint'),
       logoutUrl: config.getOptionalString('logoutUrl'),
       issuer: config.getString('issuer'),
-      cert: config.getOptionalString('cert'),
-      privateCert: config.getOptionalString('privateKey'),
+      cert: config.getString('cert'),
+      privateKey: config.getOptionalString('privateKey'),
```

Comment:
`privateCert` was deprecated, and removed, [here](https://github.com/node-saml/passport-saml/pull/591).